src/service/transaction_service.rb

```diff
@@ -0,0 +1,10 @@
+module TransactionService
+    
+    def TransactionService.sortByDate(transactions)
```

Comment:
ruby style prefers snake case to camel case
```suggestion
    def TransactionService.sort_by_date(transactions)
```